packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -609,6 +609,48 @@ export const PARAMS_AND_SECRETS_LAMBDA_LAYER_ARNS: { [version: string]: { [arch:
 };
 
 const ADOT_LAMBDA_LAYER_JAVA_SDK_ARNS: { [version: string]: { [arch: string]: { [region: string]: string } } } = {
+  '1.26.0': {
+    x86_64: {
+      'ap-northeast-1': 'arn:aws:lambda:ap-northeast-1:901920570463:layer:aws-otel-java-wrapper-amd64-ver-1-26-0:2',
```

Comment:
Is there a "Releases" page where we can see which versions have been released for each of these layers?

Reply:
I have these PRs open to update our `Releases` pages with the new Lambda Layer versions but I have not merged them in yet as I am waiting for this PR to be merged before completing the release process:
- https://github.com/aws-otel/aws-otel.github.io/pull/564
- https://github.com/aws-observability/aws-otel-lambda/pull/598